crates/websocket/src/lib.rs

```diff
@@ -28,12 +31,16 @@ impl LemmyContext {
     chat_server: Addr<ChatServer>,
     client: Client,
     activity_queue: QueueHandle,
+    settings: Settings,
+    secret: Secret,
```

Comment:
The main change in this PR. Settings and Secret are now part of the Context, rather than global singletons.